storage/innobase/include/os0file.h

```diff
@@ -185,10 +185,14 @@ class IORequest
     WRITE_SYNC= 16,
     /** Asynchronous write */
     WRITE_ASYNC= WRITE_SYNC | 1,
+    /** Asynchronous doublewritten page */
+    WRITE_DBL= WRITE_ASYNC | 4,
```

Comment:
I do not want to expose that constant outside `IORequest`. The `enum` is `public`; there is no concept of `private` values. Sure, I could add a `private` `static constexpr` for that value, but I’d say that it would obfuscate things more.